src/js/components/mon-compte/export-credentials-component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+
+class ExportCredentialsField extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = { value: 'python' };
+		this.credentials = props.credentials;
+	}
+
+	handleChange = (e) => this.setState({ value: e.target.value });
+
+	render() {
+		const pythonText = `ceci est l'export en Python : ${this.credentials.AWS_ACCESS_KEY_ID}`;
+		const RText = `ceci est l'export en R : ${this.credentials.AWS_S3_ENDPOINT}`;
```

Comment:
Instead of defining multiple variables, we can store all the inputs as a Key / Value object with key = language, value = export.  
Or write a function that accepts the language as input and outputs the export. This would be more efficient as it won't create unused exports.

Reply:
The function could also benefit from a templating feature with templates being stored as resources in other files

Reply:
I added a template file but I don't think it's the right way to do it